docs/source/conf.py

```diff
@@ -191,4 +191,7 @@ def setup(app):
 
 # @jpchen's hack to get rtd builder to install latest pytorch
 if 'READTHEDOCS' in os.environ:
-    os.system('sh scripts/install_pytorch.sh')
+    os.system('curl -o install.sh https://raw.githubusercontent.com/uber/pyro/dev/scripts/install_pytorch.sh')
```

Comment:
Could you explain why this hack is required when the same files are present locally?